### PR TITLE
Add README files 1/7

### PR DESCRIPTION
**Pull request by Leoriem-code**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: arithmetic_analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Arithmetic analysis
+
+It's a branch of mathematics that deal with solving linear equations.
```
Comment (poyea) [inline arithmetic_analysis/README.md:3]:
```suggestion
Arithmetic analysis is a branch of mathematics that deals with solving linear equations.
```
Or should we call it numerical analysis?

Comment (Leoriem-code) [reply arithmetic_analysis/README.md:3]: Now that you talk about it, I believe numerical analysis is better:

> Examples of numerical analysis include: ordinary differential equations as found in celestial mechanics, **numerical linear algebra** in data analysis, and stochastic differential equations and Markov chains for simulating living cells in medicine and biology.

Comment (Leoriem-code) [reply arithmetic_analysis/README.md:3]: But then, we would have to change the name of the folder too

File: audio_filters/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Audio Filter
+
+Audio filters work on the frequency of an audio signal to attenuate unwanted frequency and amplify wanted ones.
+They are used within anything related to sound, whether it's radio communication or a hi-fi system.
```
Comment (poyea) [inline audio_filters/README.md:4]:
```suggestion
They are used within anything related to sound, whether it is radio communication or a hi-fi system.
```

File: backtracking/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Backtracking
+
+Backtracking is a way to speed up the search process by removing candidate when they can't be the solution of a problem.
```
Comment (poyea) [inline backtracking/README.md:3]:
```suggestion
Backtracking is a way to speed up the search process by removing candidates when they can't be the solution of a problem.
```

File: bit_manipulation/README.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
-* https://docs.python.org/3/reference/expressions.html#binary-bitwise-operations
-* https://docs.python.org/3/reference/expressions.html#unary-arithmetic-and-bitwise-operations
-* https://docs.python.org/3/library/stdtypes.html#bitwise-operations-on-integer-types
-* https://wiki.python.org/moin/BitManipulation
-* https://wiki.python.org/moin/BitwiseOperators
-* https://www.tutorialspoint.com/python3/bitwise_operators_example.htm
+# Bit manipulation
+
+Bit manipulation is the act of manipulating bits to detect errors (hamming code), encrypts and decrypts messages (more on that in the 'ciphers' folder) or just do anything at the core level of your computer.
```
Comment (poyea) [inline bit_manipulation/README.md:3]:
```suggestion
Bit manipulation is the act of manipulating bits to detect errors (hamming code), encrypts and decrypts messages (more on that in the 'ciphers' folder) or just do anything at the lowest level of your computer.
```

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**